lib/schemas/payment.js

```diff
@@ -0,0 +1,93 @@
+const {config} = require('bedrock');
+const {PaymentStatus} = require('bedrock-payment');
+
+const w3cDateTime = require('bedrock-validation/schemas/w3cDateTime');
+const numericFloat = '^[0-9]+(\\.[0-9]+)?$';
```

Comment:
until we start dealing with wonky crytocurrency, seems like this should limit to two places behind decimal point.  Also, it appears to me that the decimal point is optional here, is that what is required?

Reply:
Decimal point is optional.

Also according this standard ISO_4217: http://travel.fyicenter.com/Currency/Currency_Decimal_Places_ISO_4217.php?P=3

some currencies allow 3 decimal places.

I'm not sure limiting the number of decimal places is a good idea as ISO conventions aside BigNumber should be able to handle a string with any number of decimals places.

Reply:
Do the interactions with Paypal use ISO_4217?

If an order is sent to paypal for US for '4.5555', does it work or blow up?

Reply:
GOOD CATCH SIR:

```
 "field": "/purchase_units/0/amount/value",
            "value": "14.555",
            "issue": "DECIMAL_PRECISION",
            "description": "If the currency supports decimals, only two decimal place precision is supported."
```

Reply:
Addressed here: https://github.com/digitalbazaar/bedrock-payment-http/pull/1/commits/a25c3d8ebb2119f8cb6030a615f6478cabe8e426

Reply:
I plugged this regex in here and it doesn't seem to work, maybe I'm doing something wrong?  https://regex101.com/r/hhMydW/1

It also appears that the decimal point and digits afterward are still optional, does this mean that the paypal API in particular accepts integer values for USD amounts without difficulty?

Reply:
You left in the double escape for `.`

should be: ^[0-9]+(\.[0-9]{0,2})?$
We have to double escape \\. for the json schema.